src/linter/worker.go

```diff
@@ -315,6 +315,11 @@ func (w *Worker) analyzeFile(file *workspace.File, rootNode *ir.Root) (*rootWalk
 	}
 	walker.afterLeaveFile()
 
+	if !walker.declareSection {
+		walker.Report(rootNode, LevelWarning, "noDeclareSection", "You have no declare section with strict_types=1 argument")
```

Comment:
maybe "missed `declare(strict_types=1)` directive"?

Reply:
I used a more formalized form, I agree, it could be simplified